.github/workflows/nouse_install.yml

```diff
@@ -115,13 +115,6 @@ jobs:
         run: |
           echo "github.event_name: ${{ github.event_name }}"
 
-      - name: dependency by homebrew
```

Comment:
It seems no reason to stay in python 3.10, therefore I removed homebrew step

Reply:
LGTM